lib/migration.js

```diff
@@ -348,6 +486,33 @@ function mixinMigration(MySQL, mysql) {
     }
   };
 
+  MySQL.prototype.getAlterStatement = function(model, statements) {
+    return statements.length ?
+      'ALTER TABLE ' + this.tableEscaped(model) + ' ' + statements.join(',\n') :
+      '';
+  };
+
+  MySQL.prototype.buildForeignKeyDefinition = function(model, keyName) {
+    //TODO validate that these are in the same DB, etc.
```

Comment:
Sorry. It's actually was already about 6 lines down from that. I just removed the TODO